lib/lm_docstache/parser.rb

```diff
@@ -115,6 +128,9 @@ def replace_variables_in_document!
     end
 
     def has_skippable_variable?(text)
+      hide_custom_tags.select do |pattern|
+        text =~ pattern
+      end.size > 0 ||
```

Comment:
You don't need to use `select` here, given you can stop the iterations once first case is found, so I suggest you to:
```suggestion
      return true if hide_custom_tags.find { |pattern| text =~ pattern }
```